.github/workflows/build.yml

```diff
@@ -50,10 +50,8 @@ jobs:
           docker version
           docker-compose version
 
-      - name: Log in to container registry
-        run: echo "${DOCKER_GCR_SERVICE_ACCOUNT}" | docker login -u _json_key --password-stdin https://gcr.io
-        env:
-          DOCKER_GCR_SERVICE_ACCOUNT: ${{ secrets.DOCKER_GCR_SERVICE_ACCOUNT }}
+      - name: Log into Docker Hub
+        run: echo "${{ secrets.DOCKERHUB_PASSWORD }}" | docker login -u "${{ secrets.DOCKERHUB_USERNAME }}" --password-stdin
```

Comment:
Are you using the `dockermediacloudtravis` user here and a personal access token instead of a password?

Reply:
No, I added new GitHub secrets for the `mcsystems` user in the `mc2021` Docker Hub org

Reply:
Could you either reuse `dockermediacloudtravis` user for pulling / pushing those images, or (perhaps better) add separate Docker Hub users for:

1) CI (currently GitHub Actions) pulling / pushing the images;
2) Production servers pulling the images.

All of these users should have 2FA enabled, use personal access tokens instead of their passwords, know nothing about billing, and have only appropriate permissions to pull from / push to Docker Hub (e.g. the user doing the pulling needs only read-only permissions - I don't know if that's doable on Docker Hub); we'd then be using `mcsystems` only for billing and administrative tasks.

I've made the mistake of calling `dockermediacloudtravis` user after the specific CI that we were using at the time (Travis CI) so don't mention GitHub Actions or any specific "brandnames" in the usernames :)